peachjam/migrations/0152_backfill_natures.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.14 on 2024-08-29 13:58
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    # backfill natures for existing documents
+    DocumentNature = apps.get_model("peachjam", "DocumentNature")
+
+    DOC_TYPES = [
+        ("core_document", "document", "Document"),
+        ("gazette", "gazette", "Gazette"),
+        ("generic_document", "document", "Document"),
+        ("judgment", "judgment", "Judgment"),
+        ("legal_instrument", "legal-instrument", "Legal Instrument"),
+        ("legislation", "act", "Act"),
+        ("book", "book", "Book"),
+        ("journal", "journal", "Journal"),
+    ]
+    for doc_type, nature, name in DOC_TYPES:
+        nature = DocumentNature.objects.get_or_create(
+            code=nature, defaults={"name": name}
+        )[0]
```

Comment:
![image](https://github.com/user-attachments/assets/3cf83091-5998-404a-a878-5cb15a9b3d91)
@longhotsummer I'm not sure what happened but when I ran the migration, the names were not filled in, I didn't check it before I ran the migration, so I may have had bad data already I'm not sure

Reply:
maybe get_or_create doesn't work in migrations

Reply:
i've changed the migration to set the name explicitly